src/index.html

```diff
@@ -16,7 +37,338 @@
       defer
     ></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <!-- #endregion -->
+  <body class="page__body">
+    <header class="header">
+      <div class="container">
+        <div class="top-bar">
+          <a
+            href="#"
+            class="top-bar__logo"
+          >
+            <img
+              src="./images/header/logo.png"
+              alt="Bang & Olufsen logo"
+              class="top-bar__logo-img"
+            />
+          </a>
+          <div class="top-bar__icons">
+            <a
+              href="tel:+1 234 5555-55-55"
```

Comment:
it is better to remove spaces and hyphens from phone numbers in links to avoid possible problems with number interpretation by different devices and browsers
a clean, uninterrupted digital format is best practice
`<a href="tel:+123455555555" class="icon icon--phone">`